### PR TITLE
Add clone form

### PR DESCRIPTION
**Pull request by mariarivera30**
*Issue number of the reported bug or feature request: #<number>*

**Describe your changes**
A clear and concise description of the changes you have made.

**Testing performed**
Describe the testing you have performed to ensure that the bug has been addressed, or that the new feature works as planned.

**Additional context**
Add any other context about your contribution here.

### CODE REVIEW

File: pybossa/view/projects.py [hunk 1]
```diff
@@ -429,6 +429,70 @@ def _description_from_long_description():
...
+@admin_or_subadmin_required
+def clone(short_name):
+
+    project, owner, ps = project_by_shortname(short_name)
+    project_sanitized, owner_sanitized = sanitize_project_owner(project,
+                                                                owner,
+                                                                current_user,
+                                                                ps)
+    ensure_authorized_to('read', project)
+    form = dict(name = project.short_name,
```
Comment (dchhabda) [inline pybossa/view/projects.py]: is this needed?

File: pybossa/view/projects.py [hunk 2]
```diff
@@ -429,6 +429,70 @@ def _description_from_long_description():
...
+        ensure_authorized_to('create', Project)
+
+        form = dynamic_clone_project_form(ProjectCommonForm, request.form, data_access_levels)
+
+        if not form.validate():
+            flash(gettext('Please correct the errors'), 'error')
+        else:
+            new_project = clone_project(project, form.data)
+            project_repo.save(new_project)
+            flash(gettext('Project cloned!'), 'success')
```
Comment (dchhabda) [inline pybossa/view/projects.py]: Successful cloning to be audit logged.

File: pybossa/view/projects.py [hunk 3]
```diff
@@ -429,6 +429,70 @@ def _description_from_long_description():
...
+                                                                owner,
+                                                                current_user,
+                                                                ps)
+    ensure_authorized_to('read', project)
+    form = dict(name = project.short_name,
+                short_name = project.short_name,
+                password = '',
+                copy_users= False)
+
+    form = dynamic_clone_project_form(ProjectCommonForm, None, data_access_levels, obj=project)
```
Comment (dchhabda) [inline pybossa/view/projects.py]: seems this is needed for during get request before line #489

File: pybossa/view/projects.py (hunk 1 above)
Comment (mariarivera30) [reply pybossa/view/projects.py]: No I forgot to removed it.

File: pybossa/view/projects.py (hunk 3 above)
Comment (mariarivera30) [reply pybossa/view/projects.py]: will do

File: pybossa/view/projects.py (hunk 2 above)
Comment (mariarivera30) [reply pybossa/view/projects.py]: will added it

**Pull request merged**